simulations/getting-started.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting Started
+---
+
+import { Tabs, Tab } from 'nextra/components';
+
+This guide will walk you through the basic setup required to run your first simulation and see the results in LangWatch.
+
+For more in-depth information and advanced use cases, please refer to the official [`scenario` library documentation](https://github.com/langwatch/scenario).
+
+## 1. Installation
+
+First, you need to install the `scenario` library in your project. Choose your language below.
+
+<Tabs>
+  <Tab title="Python">
+    ```bash
+    pip install scenario-langwatch
+    ```
+  </Tab>
+  <Tab title="TypeScript">
+    ```bash
+    npm install @langwatch/scenario
+    ```
+  </Tab>
+</Tabs>
+
+
+## 2. Configure Environment Variables
+
+We recommend creating a `.env` file in the root of your project to manage your environment variables.
+
+```bash title=".env"
+LANGWATCH_API_KEY="your-api-key"
+LANGWATCH_ENDPOINT="https://app.langwatch.ai"
+```
+
+You can find your `LANGWATCH_API_KEY` in your LangWatch project settings.
+
+
+## 3. Create a Basic Scenario
+
+Here's how to create and run a simple scenario to test an agent.
+
+First, you need to wrap your agent in a simple function that takes a list of messages and returns a string response.
```

Comment:
or vice versa, or both list of messages in and out, or both string in and out

point them to the integration docs: https://scenario.langwatch.ai/agent-integration